### ISSUE TITLE
Back returns to main menu in Flows

### ISSUE THREAD

**Issue by stanger89** [jusjoken/gemstone2#18]
When viewing a flow, the Back command will return to the main menu instead of the parent level.  When combined with the browsing location not being saved when leaving a flow, hitting back results in excess scrolling to return to the last browsing location.

**Comment by jptheripper** [comment]
I agree, "up" isnt just very intuitive to return to the parent level.  It should be back.

**Comment by jusjoken** [comment]
i will review the back behavior but leave the other up/left behavior as is
